internal/exportTypescriptSchema.ts

```diff
@@ -0,0 +1,34 @@
+import { DURATION_TS, TIME_TS, generateTypeScript } from "./generateTypeScript";
+import { foxgloveEnumSchemas, foxgloveMessageSchemas } from "./schemas";
+
+/**
+ * Export typescript schema as source, keyed by the schema name.
+ *
+ * @returns a record of schema name => schema source.
+ */
+export function exportTypescriptSchema(): Record<string, string> {
```

Comment:
```suggestion
/**
 * Export typescript schemas as source, keyed by the file name.
 *
 * @returns a record of schema name => schema source.
 */
export function exportTypeScriptSchemas(): Record<string, string> {
```

I would go with "file name" because `index` is not really a "schema name". This also makes me wonder if we should add `".ts"` suffix to the keys as well?

Maybe use a Map?